metadata.md

```diff
@@ -3,15 +3,25 @@ layout: default
 css_id: metadata
 ---
 
-# Metadata
+# Roles and Metadata
 
-Metadata files provide information that clients can use to make update decisions. Different metadata files provide different information. The various metadata files are signed by different keys as are indicated by the root role. The concept of roles allows TUF to only trust information that a role is trusted to provide.
+TUF uses metadata to create verifiable records about the state
+of a repository or application at a specified time and, as such, clients can use
+it to make decisions on which files to update. Different metadata files provide
+different information, and will be signed by different roles.
 
-The signed metadata files always include the time they were created and their expiration dates. This ensures that outdated metadata will be detected and that clients can refuse to accept metadata older than that which they've already seen.
+Roles define the set of actions a party can perform.The concept of roles allows
+TUF to only trust information provided by the correctly
+delegated party. The root role indicates which roles can sign for which projects.
 
-All TUF metadata uses a subset of the JSON object format. When calculating the digest of an object, we use the [Canonical JSON](http://wiki.laptop.org/go/Canonical_JSON) format. Implementation-level detail about the metadata can be found in the [spec](docs/tuf-spec.txt).
+The signed metadata files always include the time they were created and their
+expiration dates. This ensures that outdated metadata will be detected and that
+clients can refuse to accept metadata older than that which they've already seen.
 
-There are four required top-level roles and one optional top-level role, each with their own metadata file.
+All TUF metadata uses a subset of the JSON object format. When calculating the
+digest of an object, we use the [Canonical JSON](http://wiki.laptop.org/go/Canonical_JSON) format. Implementation-level detail about the metadata can be found in the [spec](docs/tuf-spec.txt).
+
```

Comment:
The link to the specification is broken.  We should use the following URL: https://github.com/theupdateframework/specification/blob/master/tuf-spec.md

Reply:
fixed